content/content-strategy/cms/optimise-cms/section3.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
!! This file is not referenced in the `index.yml` file for the 'Optimise CMS' topic. If it won't be used, please delete this file in next commit. Otherwise check that it's added at the correct location in the index.